coordinatorlib/src/main/scala/com/socrata/datacoordinator/common/SchemaFinder.scala

```diff
@@ -32,6 +29,13 @@ class SchemaFinder[CT, CV](typeSerializer: CT => TypeName, cache: Cache) extends
     val pk = ctx.pkCol_!.userColumnId
     Schema(hash, columns, pk, ctx.datasetInfo.localeName)
   }
+
+  def getSchemaWithFieldName(ctx: DatasetCopyContext[CT]): SchemaWithFieldName = {
+    val hash = schemaHash(ctx)
```

Comment:
So these functions are just being added to the library so soql-server can use them?